### PR TITLE
Add documentation for NP Atobarai

### PR DESCRIPTION
**Pull request by stnatic**
- Added a plugin manual for "business users"
- Added a dev manual explaining advanced features of the integration

### CODE REVIEW

**Comment by mateuszgrzyb** [comment]
There should be an NP Atobarai entry on
`/developer/extending/plugins#built-in-plugins`

**Review by mateuszgrzyb — APPROVED** [review]

File: docs/developer/available-plugins/np-atobarai.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+      }
+    ]
+  }
+}
+```
+
+The raw gateway codes returned by Saleor are prefixed by the operation name, e.g. `TR#` for transaction registration
+which is performed when the `checkoutComplete` mutation is invoked.
+
+For an up to date list of NP error codes please refer to the NP documentation.
```
Comment (krzysztofwolski) [inline docs/developer/available-plugins/np-atobarai.mdx:141]: Would be useful to link the docs

File: docs/developer/available-plugins/np-atobarai.mdx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- *SALEORNP008* - carrier company code is invalid
+
+For errors that cannot be parsed, we recommend to display a generic payment error.
+
+### Invalid address
+
+This is an example response when attempting to complete a checkout with an address that is not specific enough for NP.
+
+```json
+{"checkoutComplete": {"order": null, "errors": [{"field": null, "message": "TR#E0100065"}]}}
```
Comment (krzysztofwolski) [inline docs/developer/available-plugins/np-atobarai.mdx:163]: Please format the responses

File: docs/developer/available-plugins/np-atobarai.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+## Using NP Atobarai
+
+### Preparing a checkout
+
+To prepare a checkout, you need to follow the steps described in
+[Creating a checkout session](developer/checkout.mdx#creating-a-checkout-session) and
+[Selecting a shipping method](developer/checkout.mdx#selecting-a-shipping-method).
+
+Please note that NP Atobarai is only applicate to the japanese market, as it requires both a Japanese postal
+address as well as a Japanese phone number (country code: `+81`) and only supports payments in Japanese yen (`JPY`).
```
Comment (krzysztofwolski) [inline docs/developer/available-plugins/np-atobarai.mdx:19]:
```suggestion
Please note that NP Atobarai is only applicating to the Japanese market, as it requires:
- a Japanese postal address
- a Japanese phone number (country code: `+81`) 
- only supports payments in Japanese yen (`JPY`)
- the shipment has to be fulfilled using one of the supported local carriers
```

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: docs/developer/available-plugins/np-atobarai.mdx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+      message
+      code
+    }
+  }
+}
+```
+
+If the credit check was successful, an order will be created.
+
+In case of errors, NP recommends to display an error according to the communication guidelines available in their documentation.
```
[the remark this answers is not in the document]
Comment (stnatic) [reply docs/developer/available-plugins/np-atobarai.mdx:114]: It's password protected

File: docs/developer/available-plugins/np-atobarai.mdx (hunk 1 above)
Comment (stnatic) [reply docs/developer/available-plugins/np-atobarai.mdx:141]: It's password protected

**Comment by stnatic** [comment]
SALEOR-4968

**Review by patrys — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**